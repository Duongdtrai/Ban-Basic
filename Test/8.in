2
41
766 386 190 341 875 190 981 380 198 469 29 926 124 745 492 849 823 942 294 988 260 589 87 268 569 893 484 702 385 128 57 102 846 779 152 921 230 235 199 583 349 
4
244 420 761 763 
