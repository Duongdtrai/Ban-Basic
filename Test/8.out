17028633252
4123098
